### ISSUE TITLE
[Feature Request] Availability in Azure Government Cloud

### ISSUE THREAD

**Issue by nathanhannig** [microsoft/azure-load-testing#96]
I saw that Azure Load testing is currently in Preview on Azure Commercial.

It looks like a great tool that would benefit Azure Government Cloud as well.

Is it on the future roadmap to bring to Azure Government Cloud?

Thank you.

**Comment by Nagarjuna-Vipparthi** [comment]
Yes, availability on government clouds is on our future roadmap. However, the focus currently is on expanding to some more regions on the public cloud.

**Comment by nathanhannig** [comment]
This is now available and can be closed?
